setup.py

```diff
@@ -296,12 +296,7 @@ def write_version(filename: str = os.path.join(*["airflow", "git_version"])):
              druid + pinot + segment + snowflake + elasticsearch +
              atlas + azure + aws + salesforce + cgroups + papermill + virtualenv)
 
-# Snakebite & Google Cloud Dataflow are not Python 3 compatible :'(
-if PY3:
-    devel_ci = [package for package in devel_all if package not in
-                ['snakebite>=2.7.8', 'snakebite[kerberos]>=2.7.8']]
-else:
-    devel_ci = devel_all
+devel_ci = devel_all
```

Comment:
In this case, we can drop the `devel_ci` altogether, right?

Reply:
Correct - we could!

Reply:
I think we should drop it ;)

Reply:
Dropped.